[2024-12-19T10:19:52.947+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T10:19:52.964+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag_v5.get_name scheduled__2021-11-18T00:00:00+00:00 [queued]>
[2024-12-19T10:19:52.974+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag_v5.get_name scheduled__2021-11-18T00:00:00+00:00 [queued]>
[2024-12-19T10:19:52.975+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T10:19:52.995+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2021-11-18 00:00:00+00:00
[2024-12-19T10:19:53.005+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6131) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T10:19:53.008+0000] {standard_task_runner.py:72} INFO - Started process 6134 to run task
[2024-12-19T10:19:53.008+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_operator_dag_v5', 'get_name', 'scheduled__2021-11-18T00:00:00+00:00', '--job-id', '1787', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag.py', '--cfg-path', '/tmp/tmpx2oatkne']
[2024-12-19T10:19:53.012+0000] {standard_task_runner.py:105} INFO - Job 1787: Subtask get_name
[2024-12-19T10:19:53.085+0000] {task_command.py:467} INFO - Running <TaskInstance: python_operator_dag_v5.get_name scheduled__2021-11-18T00:00:00+00:00 [running]> on host 02d979add40a
[2024-12-19T10:19:53.212+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='python_operator_dag_v5' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-18T00:00:00+00:00'
[2024-12-19T10:19:53.217+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T10:19:53.284+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-19T10:19:53.297+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T10:19:53.302+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_operator_dag_v5, task_id=get_name, run_id=scheduled__2021-11-18T00:00:00+00:00, execution_date=20211118T000000, start_date=20241219T101952, end_date=20241219T101953
[2024-12-19T10:19:53.353+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T10:19:53.386+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T10:19:53.389+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
